docs/docs/tutorial/chapter2/getting-dynamic.md

```diff
@@ -21,7 +21,10 @@ We use [Prisma](https://www.prisma.io/) to talk to the database. Prisma has anot
 
 First let's define the data structure for a post in the database. Open up `api/db/schema.prisma` and add the definition of our Post table (remove any "sample" models that are present in the file, like the `UserExample` model). Once you're done, the entire schema file should look like:
 
-```javascript title="api/db/schema.prisma"
+<Tabs groupId="sql-mongo">
+<TabItem value="sql" label="SQLite">
+
+```Javascript title="api/db/schema.prisma"
```

Comment:
It looks like docusaurus doesn't recognize "Javascript". Lowercasing it again should bring back syntax highlighting
```suggestion
```javascript title="api/db/schema.prisma"
```